### ISSUE TITLE
papers

### ISSUE THREAD

**Issue by usefulhyun** [usefulhyun/llm-mt-papers#1]
-  Exploring Human-Like Translation Strategy with Large Language Models
- https://github.com/zwhe99/MAPS-mt
- Synthetic Data Generation with Large Language Models for Text Classification: Potential and Limitations
- https://openreview.net/forum?id=TtQfZwf5s5&noteId=wiBH1s1MXU
- Recent Advances in Interactive Machine Translation with Large Language Models
- Synthetic Data Generation and Joint Learning for Robust Code-Mixed Translation
- From LLM to NMT: Advancing Low-Resource Machine Translation with Claude
- Multilingual Machine Translation with Large Language Models: Empirical Results and Analysis
- Steering Large Language Models for Machine Translation with Finetuning and In-Context Learning
- https://learn.microsoft.com/en-us/globalization/localization/ai/ai-and-llms-for-translation
- How Effective is Synthetic Data and Instruction Fine-tuning for Translation with Markup using LLMs?
- https://ai.meta.com/blog/how-to-fine-tune-llms-peft-dataset-curation/
- Improving Translation Quality with Domain-Specific Fine-Tuning and NVIDIA NIM

**Comment by usefulhyun** [comment]
- https://www.amazon.science/blog/using-large-language-models-llms-to-synthesize-training-data
- From General LLM to Translation: How we dramatically improve translation quality using human evaluation data for LLM finetuning

- Chain-of-Dictionary Prompting Elicits Translation in Large Language Models
- Towards Effective Disambiguation for Machine Translation with Large Language Models
- Fine-tuning Large Language Models for Adaptive Machine Translation
-

**Comment by usefulhyun** [comment]
- [Contextual Refinement of Translations: Large Language Models for Sentence and Document-Level Post-Editing](https://arxiv.org/abs/2310.14855). Sai Koneru, Miriam Exel, Matthias Huck, Jan Niehues. (NAACL 2024)
- [Encouraging Divergent Thinking in Large Language Models through Multi-Agent Debate](https://arxiv.org/abs/2305.19118). Tian Liang, Zhiwei He, Wenxiang Jiao, Xing Wang, Yan Wang, Rui Wang, Yujiu Yang, Zhaopeng Tu, Shuming Shi. (arxiv 2023) {[code](https://github.com/Skytliang/Multi-Agents-Debate)} (EMNLP 2024)
- [Leveraging GPT-4 for Automatic Translation Post-Editing](https://arxiv.org/abs/2305.14878). Vikas Raunak, Amr Sharaf, Hany Hassan Awadallah, Arul Menezes. (EMNLP findings 2023)
- [Lost in the Source Language: How Large Language Models Evaluate the Quality of Machine Translation](https://arxiv.org/abs/2401.06568). Xu Huang, Zhirui Zhang, Xiang Geng, Yichao Du, Jiajun Chen, Shujian Huang. (ACL findings 2024) {[code](https://github.com/xuuHuang/lost_in_the_src)}
- [Error Analysis Prompting Enables Human-Like Translation Evaluation in Large Language Models: A Case Study on ChatGPT](https://arxiv.org/abs/2303.13809). Qingyu Lu, Baopu Qiu, Liang Ding, Liping Xie, Dacheng Tao. (ACL findings 2024) {[code](https://github.com/Coldmist-Lu/ErrorAnalysis_Prompt)}
- [Large Language Models Are State-of-the-Art Evaluators of Translation Quality](https://aclanthology.org/2023.eamt-1.19/). Tom Kocmi, Christian Federmann (EAMT 2023) {[code](https://github.com/MicrosoftTranslator/GEMBA)}

- [Aligning Translation-Specific Understanding to General Understanding in Large Language Models](https://arxiv.org/abs/2401.05072). Yichong Huang, Xiaocheng Feng, Baohang Li, Chengpeng Fu, Wenshuai Huo, Ting Liu, Bing Qin (EMNLP 2024)
- [TIM: Teaching Large Language Models to Translate with Comparison](https://arxiv.org/abs/2307.04408). Jiali Zeng, Fandong Meng, Yongjing Yin, Jie Zhou. (AAAI 2024) {[code](https://github.com/lemon0830/TIM)}
- [Eliciting the Translation Ability of Large Language Models via Multilingual Finetuning with Translation Instructions](https://arxiv.org/abs/2305.15083). Jiahuan Li, Hao Zhou, Shujian Huang, Shanbo Chen, Jiajun Chen. (TACL 2024)
- [Prompting Large Language Model for Machine Translation: A Case Study](https://openreview.net/pdf?id=yWl0agiI0y). Biao Zhang, Barry Haddow, Alexandra Birch. (ICML 2023)
- [Prompting Neural Machine Translation with Translation Memories](https://arxiv.org/abs/2301.05380). Abudurexiti Reheman, Tao Zhou, Yingfeng Luo, Di Yang, Tong Xiao, Jingbo Zhu. (AAAI 2023)
- [Adaptive Machine Translation with Large Language Models](https://arxiv.org/abs/2301.13294). Yasmin Moslem, Rejwanul Haque, John D. Kelleher, Andy Way. (EAMT 2023) {[code](https://github.com/ymoslem/Adaptive-MT-LLM)}
- [In-context Examples Selection for Machine Translation](https://aclanthology.org/2023.findings-acl.564.pdf). Sweta Agrawal, Chunting Zhou, Mike Lewis, Luke Zettlemoyer, Marjan Ghazvininejad. (Findings of the ACL 2023)

- [The unreasonable effectiveness of few-shot learning for machine translation](https://openreview.net/pdf?id=zvCSNsoyKW). Xavier Garcia, Yamini Bansal, Colin Cherry, George Foster, Maxim Krikun, Fangxiaoyu Feng, Melvin Johnson, Orhan Firat. (ICML 2023)
- [Dictionary-based Phrase-level Prompting of Large Language Models for Machine Translation](https://arxiv.org/abs/2302.07856). Marjan Ghazvininejad, Hila Gonen, Luke Zettlemoyer. (arxiv 2023)

- [RAMP: Retrieval and Attribute-Marking Enhanced Prompting for Attribute-Controlled Translation](https://aclanthology.org/2023.acl-short.126/). Gabriele Sarti, Phu Mon Htut, Xing Niu, Benjamin Hsu, Anna Currey, Georgiana Dinu, Maria Nadejde. (ACL 2023) 

- [Domain Terminology Integration into Machine Translation: Leveraging Large Language Models](https://arxiv.org/abs/2310.14451). Yasmin Moslem, Gianfranco Romani, Mahdi Molaei, Rejwanul Haque, John D. Kelleher, Andy Way. (WMT 2023)

- [Towards Robust In-Context Learning for Machine Translation with Large Language Models](https://aclanthology.org/2024.lrec-main.1444/). Shaolin Zhu, Menglong Cui, Deyi Xiong. (LREC-COLING 2024)

- [Towards Effective Disambiguation for Machine Translation with Large Language Models](https://arxiv.org/abs/2309.11668). Vivek Iyer, Pinzhen Chen, Alexandra Birch. (WMT 2023)

- [Instruction Position Matters in Sequence Generation with Large Language Models](https://arxiv.org/abs/2308.12097). Yijin Liu, Xianfeng Zeng, Fandong Meng, Jie Zhou. (arxiv 2023) {[code](https://github.com/Adaxry/Post-Instruction/tree/main)}

**Comment by usefulhyun** [comment]
- [Document-Level Machine Translation with Large Language Models](https://aclanthology.org/2023.emnlp-main.1036/) (Wang et al., EMNLP 2023)

- [Ladder: A Model-Agnostic Framework Boosting LLM-based Machine Translation to the Next Level](https://aclanthology.org/2024.emnlp-main.860/) (Feng et al., EMNLP 2024)

- [MT-PATCHER: Selective and Extendable Knowledge Distillation from Large Language Models for Machine Translation](https://aclanthology.org/2024.naacl-long.358/) (Li et al., NAACL 2024)

- [When Is Multilinguality a Curse? Language Modeling for 250 High- and Low-Resource Languages](https://aclanthology.org/2024.emnlp-main.236/) (Chang et al., EMNLP 2024)
- [Towards Cross-Cultural Machine Translation with Retrieval-Augmented Generation from Multilingual Knowledge Graphs](https://aclanthology.org/2024.emnlp-main.914/) (Conia et al., EMNLP 2024)


- [Aya Model: An Instruction Finetuned Open-Access Multilingual Language Model](https://aclanthology.org/2024.acl-long.845/) (Üstün et al., ACL 2024)

- [Dolma: an Open Corpus of Three Trillion Tokens for Language Model Pretraining Research](https://aclanthology.org/2024.acl-long.840/) (Soldaini et al., ACL 2024)

- [MultiLegalPile: A 689GB Multilingual Legal Corpus](https://aclanthology.org/2024.acl-long.805/) (Niklaus et al., ACL 2024)

- [Do Llamas Work in English? On the Latent Language of Multilingual Transformers](https://aclanthology.org/2024.acl-long.820/) (Wendler et al., ACL 2024)

- [G-DIG: Towards Gradient-based DIverse and hiGh-quality Instruction Data Selection for Machine Translation](https://aclanthology.org/2024.acl-long.821/) (Pan et al., ACL 2024)

- [IndicLLMSuite: A Blueprint for Creating Pre-training and Fine-Tuning Datasets for Indian Languages](https://aclanthology.org/2024.acl-long.843/) (Khan et al., ACL 2024)

- [MultiPICo: Multilingual Perspectivist Irony Corpus](https://aclanthology.org/2024.acl-long.849/) (Casola et al., ACL 2024)

- [A Pretrainer’s Guide to Training Data: Measuring the Effects of Data Age, Domain Coverage, Quality, & Toxicity](https://aclanthology.org/2024.naacl-long.179/) (Longpre et al., NAACL 2024)


-

**Comment by usefulhyun** [comment]
https://openreview.net/forum?id=0NMzBwqaAJ
